ghost/core/test/e2e-api/members-comments/comments.test.js

```diff
@@ -381,7 +381,8 @@ describe('Comments API', function () {
                     .expectStatus(201)
                     .matchHeaderSnapshot({
                         etag: anyEtag,
-                        location: anyLocationFor('comments')
+                        location: anyLocationFor('comments'),
+                        'x-cache-invalidate': matchers.anyString
```

Comment:
This is annoying, and could potentially be improved.

We are using a dynamically generated comment, rather than a fixture, which means the value of teh header changes for each test run